src/Core/Utilities/ServiceCollectionExtensions.cs

```diff
@@ -184,6 +184,18 @@ public static void AddDefaultServices(this IServiceCollection services, GlobalSe
             // Required for UserService
             services.AddWebAuthn(globalSettings);
 
+            services.AddSingleton<IStripeAdapter, StripeAdapter>();
+            services.AddSingleton<Braintree.IBraintreeGateway>((serviceProvider) =>
```

Comment:
I had discussed the possibility of doing this on #1454. I think this is a really good idea!